### DISCUSSION TITLE
Is there a way to load a network only uising .xml file, i.e., without .bin file?

### DISCUSSION THREAD

**Discussion by yujiepan-work** [openvinotoolkit/openvino#19207] [Q&A]
Hi, currently the following codes in Python can load a model:
```python
ie = IECore()
net = ie.read_network(model=path_to_xml_file, weights=path_to_bin_file)
```
Is there a way to load a randomly-initialized model with only the xml file? Or  can I manually create a (random) bin file?

**Comment by ilya-lavrenov** [comment]
Bin file contain weights for the model.
If you model has such weights - they will be loaded automatically
If model does not require weights, you don't have to have such file
